src/components/new/FundInfo.js

```diff
@@ -3,43 +3,58 @@ import { Box, Input, LinearProgress, TextField } from "@material-ui/core";
 import RazorpayPayment from "../RazorpayPayment";
 import { formatCurrency } from "./TopDonors";
 
-const FundInfo = ({ schoolInformation, raisedAmount, requiredAmount, progress, orgCode, href, placeid }) => {
-  
-  const {schoolName , schoolAdress} = schoolInformation
+const FundInfo = ({
+  schoolInformation,
+  raisedAmount,
+  requiredAmount,
+  progress,
+  orgCode,
+  href,
+  placeid,
+}) => {
+  const { schoolName, schoolAdress } = schoolInformation;
   const [name, setName] = useState("");
   const [email, setEmail] = useState("");
   const [amount, setAmount] = useState();
-  const [updateRaiseAmount,setUpdateRaisedAmount] = useState(false)
+  const [updateRaiseAmount, setUpdateRaisedAmount] = useState(false);
   return (
     <div className="fund-info">
       <div className="fund-title new-classname-for-funds">
         <h1 style={{ textAlign: "center" }} className="fund-name">
           {schoolName}
         </h1>
         <div className="fund-links">
-          <a>
-            <img src="/new-assets/share-icon.svg" />
+          
+          <a
+            href={`whatsapp://send?text=Help me to Support this campaign ${window.location.href.replace('http://localhost:3000' , 'https://vidyartha.org')}`}
```

Comment:
@OMGATE23 
build is failing here, window cant be accessed like this,  please use router